CITATION.cff

```diff
@@ -15,8 +15,9 @@ authors:
   orcid: "https://orcid.org/0000-0002-2047-1649"
 title: "SRAI: Spatial Representations for Artificial Intelligence"
 version: 0.6.1
-date-released: 2022-11-23
-url: "https://kraina-ai.github.io/srai"
+doi: 10.1145/3615886.3627740
```

Comment:
I think we could add a conference-paper citation now. See this example from HF repo https://github.com/huggingface/datasets/blob/main/CITATION.cff